kuksa-sdk/src/main/kotlin/org/eclipse/kuksa/extension/DataPointExtension.kt

```diff
@@ -54,6 +56,8 @@ val <T : Any> VssSignal<T>.datapoint: Datapoint
             IntArray::class -> ValueCase.INT32_ARRAY.createDatapoint(stringValue)
             BooleanArray::class -> ValueCase.BOOL_ARRAY.createDatapoint(stringValue)
             LongArray::class -> ValueCase.INT64_ARRAY.createDatapoint(stringValue)
+            FloatArray::class -> ValueCase.FLOAT_ARRAY.createDatapoint(stringValue)
+            UIntArray::class -> ValueCase.UINT32_ARRAY.createDatapoint(stringValue)
 
             else -> throw IllegalArgumentException("Could not create datapoint for the value class: ${value::class}!")
```

Comment:
```suggestion
            else -> throw IllegalArgumentException("Could not create datapoint for the dataType class: ${dataType::class}!")
```

should be changed to dataType